enterprise/frontend/test/metabase-enterprise/sandboxes/sandboxes.cy.spec.js

```diff
@@ -574,5 +574,200 @@ describeWithToken("formatting > sandboxes", () => {
+              },
+              type: "query",
+            },
+            display: "table",
+            visualization_settings: {},
+          }).then(({ body: { id: CARD_ID } }) => {
+            cy.log(
+              "**-- 2. Sandbox `Orders` table based on previous question --**",
+            );
+
+            cy.request("POST", "/api/mt/gtap", {
+              attribute_remappings: {
+                user_id: [
+                  "dimension",
+                  ["joined-field", ORDERS_ALIAS, ["field-id", ORDERS.USER_ID]],
+                ],
+              },
+              card_id: CARD_ID,
+              group_id: COLLECTION_GROUP_ID,
+              table_id: ORDERS_ID,
+            });
+
+            cy.log(
+              "**-- 3. Sandbox `Products` table based on previous question --**",
+            );
+
+            cy.request("POST", "/api/mt/gtap", {
+              attribute_remappings: {
+                user_id: [
+                  "dimension",
+                  [
+                    "joined-field",
+                    REVIEWS_ALIAS,
+                    ["field-id", REVIEWS.PRODUCT_ID],
+                  ],
+                ],
+              },
+              card_id: CARD_ID,
+              group_id: COLLECTION_GROUP_ID,
+              table_id: PRODUCTS_ID,
+            });
+
+            cy.log("**-- 4. Fetch permissions graph --**");
+
+            cy.request("GET", "/api/permissions/graph", {}).then(
+              ({ body: { groups, revision } }) => {
+                // Update permissions for `collections` group [id: 4]
+                // This mutates the original `groups` object => we'll pass it next to the `PUT` request
+                groups[COLLECTION_GROUP_ID] = {
+                  1: {
+                    schemas: {
+                      PUBLIC: {
+                        [PRODUCTS_ID]: { query: "segmented", read: "all" },
+                        [ORDERS_ID]: { query: "segmented", read: "all" },
+                      },
+                    },
+                  },
+                };
+
+                cy.log("**-- 5. Update/save permissions --**");
+
+                cy.request("PUT", "/api/permissions/graph", {
+                  groups,
+                  revision,
+                });
+              },
+            );
+
+            signOut();
+
+            cy.log("**-- Logging in as sandboxed user --**");
+            cy.request("POST", "/api/session", {
+              username: sandboxed_user.email,
+              password: sandboxed_user.password,
+            });
+
+            // Go straight to orders table in custom questions
+            cy.visit("/question/new?database=1&table=2&mode=notebook");
+
+            // `Orders` join `Products`
+            cy.findByText("Join data").click();
+            popover()
+              .contains(/Products?/i)
+              .click();
+
+            // Where `Orders.User ID` = `Products.ID`
+            popover()
+              .contains("User ID")
```

Comment:
```suggestion
            // Where `Orders.Product_ID` = `Products.ID`
            popover()
              .contains("Product ID")
```

Reply:
Changed locally and pushed in https://github.com/metabase/metabase/pull/13902/commits/7570799cb9a324618081b18d91b100646572e424.